apps/vscode/src/appdev/devServerGuard.cjs

```
@@ -0,0 +1,139 @@
+});
+
+let dying = false;
+
+/**
+ * Fells the guard's own process tree, guard included — total by design:
+ * the tether dropped, so nothing here may survive.
+ */
+function die() {
+	if (dying) return;
+	dying = true;
+	if (process.platform === 'win32') {
```

ESLint

[error] 'process' is not defined.

(no-undef)